### DISCUSSION TITLE
Exporting Section Data with Soil Layers

### DISCUSSION THREAD

**Discussion by jawansel** [gempy-project/gempy#843] [Q&A]
Hi,

I'm planning to create a large number of geological sections in GemPy for slope calculations. My model will be built by importing soil layer data from an Excel file and a topographic map. Based on the tutorial [here](https://gempy-project.github.io/temp_gp3_docs/tutorials/ch1_fundamentals/ch1_3b_cross_sections.html#sphx-glr-tutorials-ch1-fundamentals-ch1-3b-cross-sections-py), I see it's possible to generate a grid for sections.

I'm assuming there's a way to access the data for the plotted soil layers. Could you guide me on how to extract this layer data from the sections? Additionally, I need to export this data – could you recommend a suitable format for this purpose, e.g DXF, and provide some guidance or examples on how to do this?

Thanks!

**Comment by NilsChudalla** [comment]
Hi @jawansel, 

that is a great application example. I see your reply is a bit older, so I am not sure if you solved your issue or not. Either way, it is not a purely gempy related issue. I would advice you look into the gempy basics of building a simple model without your "real life data" first. Once you understood the required input information, you can get into extracting that information. 

I am having my eye on this now, so feel free to revive this question :)
